### PR TITLE
Switch to openvswitch firewall driver

### PR DESCRIPTION
**Pull request by smalleni**
This gives us better performance.

### CODE REVIEW

File: roles/overcloud-deploy/templates/overcloud-deploy.sh.j2 [hunk 1]
```diff
@@ -8,7 +8,7 @@ fi
...
+{ time openstack overcloud deploy --templates /usr/share/openstack-tripleo-heat-templates/ -r /home/stack/templates/roles_data-notelem.yaml -e /home/stack/templates/scheduler-hints.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/network-isolation.yaml -e /home/stack/templates/network-environment.yaml -e /home/stack/templates/{{cephstorage_type}}-storage-environment.yaml -e /home/stack/templates/disable-notifications.yaml -e /home/stack/templates/environments/neutron-firewall.yaml-e /home/stack/templates/environments/args.yaml -e /home/stack/templates/environments/compute-params.yaml -e /home/stack/templates/environments/controller-params.yaml -e /home/stack/templates/environments/firstboot-env.yaml -e /home/stack/templates/deploy.yaml --ntp-server {{ntp_server}} 2>&1 | tee -a ${log} ; } 2>> ${log}
```
Comment (mbruzek) [inline roles/overcloud-deploy/templates/overcloud-deploy.sh.j2]: Looks like we need a space between "neutron-firewall.yaml" and "-e" here.

**Comment by smalleni** [comment]
@mbruzek http://post-office.corp.redhat.com/archives/aos-scalability/2018-July/msg00073.html

**Comment by jeremyeder** [comment]
Do we need this or do we automatically get it with Kuryr and OSP13?

**Comment by smalleni** [comment]
Kuryr will override the firewall driver to openvswitch anyway from what I understand. But having this defined will be harmless, and useful when kuryr is not used etc.

**Comment by akrzos** [comment]
@smalleni Can you squash and rebase, also we need the vars set for the osp13 deploy command.  Thanks!

**Comment by akrzos** [comment]
@smalleni We also will need this to be parameterized if it can not be changed easily post deployment.  Please advise if it can be changed post deployment, if not I can help you with the parameters so it works in our Jenkins job.

**Pull request closed without merging**